e2e/tests/functional/plugins/plot/overlayPlot.e2e.spec.js

```diff
@@ -60,6 +60,75 @@ test.describe('Overlay Plot', () => {
         expect(color).toBe('rgb(255, 166, 61)');
     });
 
+    test('Limit lines persist when series is moved to another Y Axis and on refresh', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/6338'
+        });
+        // Create an Overlay Plot with a default SWG
+        const overlayPlot = await createDomainObjectWithDefaults(page, {
+            type: "Overlay Plot"
+        });
+
+        const swgA = await createDomainObjectWithDefaults(page, {
+            type: "Sine Wave Generator",
+            parent: overlayPlot.uuid
+        });
+
+        await page.goto(overlayPlot.url);
+
+        // Assert that no limit lines are shown by default
+        await page.waitForSelector('.js-limit-area', { state: 'attached' });
+        const limitLineCount = await page.locator('.c-plot-limit-line').count();
```

Comment:
Same applies here. There is a scenario which would prefer a css locator which may not be particularly descriptive